.changeset/bright-chefs-train.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Why was this changeset empty? It should have been a patch, correct?

Because of this, we don't have a changeset entry about the upgrade, which is misleading.

 - https://github.com/FuelLabs/fuels-ts/releases/tag/v0.85.0

Reply:
For example:

```
---
"@fuel-ts/abi-typegen": patch
"@fuel-ts/forc": patch
"@fuel-ts/versions": patch
---

chore: upgrade `forc@0.56.1`
```

Reply:
I [added the entry](https://github.com/FuelLabs/fuels-ts/releases/tag/v0.85.0) manually.

![edited](https://github.com/FuelLabs/fuels-ts/assets/26660/98e923c9-3445-4151-82b6-8a8613934d89)